pkg/integrations/v4/logs/cfg.go

```diff
@@ -88,17 +89,18 @@ type YAML struct {
 
 // LogCfg logging integration config from customer defined YAML.
 type LogCfg struct {
-	Name       string            `yaml:"name"`
-	File       string            `yaml:"file"`        // ...
-	MaxLineKb  int               `yaml:"max_line_kb"` // Setup the max value of the buffer while reading lines.
-	Systemd    string            `yaml:"systemd"`     // ...
-	Pattern    string            `yaml:"pattern"`
-	Attributes map[string]string `yaml:"attributes"`
-	Syslog     *LogSyslogCfg     `yaml:"syslog"`
-	Tcp        *LogTcpCfg        `yaml:"tcp"`
-	Fluentbit  *LogExternalFBCfg `yaml:"fluentbit"`
-	Winlog     *LogWinlogCfg     `yaml:"winlog"`
-	Winevtlog  *LogWinevtlogCfg  `yaml:"winevtlog"`
+	Name           string            `yaml:"name"`
+	File           string            `yaml:"file"`        // ...
+	MaxLineKb      int               `yaml:"max_line_kb"` // Setup the max value of the buffer while reading lines.
+	Systemd        string            `yaml:"systemd"`     // ...
+	Pattern        string            `yaml:"pattern"`
+	Attributes     map[string]string `yaml:"attributes"`
+	Syslog         *LogSyslogCfg     `yaml:"syslog"`
+	Tcp            *LogTcpCfg        `yaml:"tcp"`
+	Fluentbit      *LogExternalFBCfg `yaml:"fluentbit"`
+	Winlog         *LogWinlogCfg     `yaml:"winlog"`
+	Winevtlog      *LogWinevtlogCfg  `yaml:"winevtlog"`
+	TargetFilesCnt int
```

Comment:
Small nit, can we make this new field not public? It should not be used outside this package